src/mimefactory.rs

```diff
@@ -133,11 +133,7 @@ impl<'a> MimeFactory<'a> {
     ) -> Result<MimeFactory<'a>> {
         let chat = Chat::load_from_db(context, msg.chat_id).await?;
 
-        let from_addr = context
-            .get_config(Config::ConfiguredAddr)
-            .await?
-            .unwrap_or_default();
-
+        let from_addr = context.get_configured_addr().await.unwrap_or_default();
```

Comment:
same semantic change here: we probably do not want to push an empty string as a recipient or use it as `To:`, we should bubble up all errors:

```suggestion
        let from_addr = context.get_configured_addr().await?;
```